### DISCUSSION TITLE
medianav testmenu problem

### DISCUSSION THREAD

**Discussion by peace1313** [cedricp/ddt4all#978] [Q&A]
Hi

My vehicle is dacia duster extreme 2023. 
My medianav version is 9.0.10.2.1154_r6.

Trying to enter mediav's test menu. 

i try  this password: 9911 OK, ****, 7739, OK but i get password invalid error.

It was read in a forum that the test menu was closed in the latest version
[https://www.dacianer.de/threads/firmware-6-0-10-2.51369/page-6](https://www.dacianer.de/threads/firmware-6-0-10-2.51369/page-6)

[https://www.msieurlolo.fr/forum/index.php?topic=70638.210](https://www.msieurlolo.fr/forum/index.php?topic=70638.210)

[https://www.daciaclub.cz/forum-tema/novy-upgrade-mn4-novy-dizajn-a-welcome-screen-183650?kols=4](https://www.daciaclub.cz/forum-tema/novy-upgrade-mn4-novy-dizajn-a-welcome-screen-183650?kols=4)

and there they said that I could activate it with the ddt4all application.

how can I do that ?
Thank you from now

**Comment by tatanka1969** [comment]
Seguo

**Reply by tatanka1969** [reply]
Me too

**Comment by KarelSvo** [comment]
> how can I do that ?

Which type of vehicle?
Where is the Pyren dump from your medianav?

**Reply by tatanka1969** [reply]
I have a Dacia Duster Extreme 2023
